Header header
geometry_msgs/Vector3 accelerometer
geometry_msgs/Vector3 compass
geometry_msgs/Vector3 gyro
geometry_msgs/Quaternion orientation_quat
um6_imu/EulerOrientation orientation_euler
float64[36] covariance

